### DISCUSSION TITLE
Community call to have random Icebreaker style calls

### DISCUSSION THREAD

**Discussion by eddiejaoude** [EddieHubCommunity/support#2857] [Ideas]
It would be great to have a bot command, that can take everyone in a voice channel...

- creates new voice channels (1 channel for every 2 people)
- divides people among the channels randomly
- after 3 mins brings them back into the main voice channel with everyone
- exclude certain roles (MC of starting voice channel)

**Challenges**
- moderation of so many voice channels

**Comment by naomi-lgbt** [comment]
Is this something you want to add to EddieBot? Or do we want a separate bot for this?

**Reply by eddiejaoude** [reply]
Good question, maybe a separate bot focusing on this feature only

**Reply by vanshika-srivastava** [reply]
Love it, if this event goes on for 1 hour, each person will meet 20 people and I think its superb for networking !!

**Reply by vanshika-srivastava** [reply]
When are we planning for this one ? @eddiejaoude

**Reply by eddiejaoude** [reply]
hopefully soon, I think to do it manually would not be so good. Let's see how long it takes us to put a bot together to help manage the event

We then can do a small test event, if all goes well, I will organise with our partner communities a much bigger event 🔥

**Reply by vanshika-srivastava** [reply]
Makes sense !! Definitely a bot will help - manual will be tough to follow up. You all are amazing !!

Super excited to see how things will turn out !!! We got this !!!

**Comment by eddiejaoude** [comment]
Moderation of so many channels could be difficult  - I will add this to the main description, but good to get people's thoughts below too

**Comment by eddiejaoude** [comment]
Project started https://github.com/EddieHubCommunity/EddieBotBreakout
